config/management/operational/src/json_rpc.rs

```diff
@@ -36,9 +38,24 @@ impl JsonRpcClientWrapper {
     }
 
     pub fn account_state(&self, account: AccountAddress) -> Result<AccountState, Error> {
-        self.client
-            .get_account_state(account, None)
-            .map_err(|e| Error::JsonRpcReadError("account-state", e.to_string()))
+        let f = || -> Result<AccountState, Box<dyn std::error::Error>> {
+            let account_state = self
+                .client
+                .get_account_state_with_proof(account, None, None)?
+                .into_inner();
+
+            let blob = account_state
+                .blob
+                .ok_or_else(|| {
+                    Error::JsonRpcReadError("account-state", "No Validator set".to_string())
+                })?
+                .into_bytes()?;
+            let account_state_blob = AccountStateBlob::from(bcs::from_bytes::<Vec<u8>>(&blob)?);
+            let account_state = AccountState::try_from(&account_state_blob)?;
+            Ok(account_state)
+        };
+
+        f().map_err(|e| Error::JsonRpcReadError("account-state", e.to_string()))
```

Comment:
This was me being a bit lazy and not wanting to do map_err on every single call (since there's no From impl on the error type). let me make this a bit cleaner